molgenis-metadata-manager/src/main/frontend/src/main.js

```diff
@@ -1,6 +1,3 @@
-import 'es6-promise/auto'
-import 'babel-polyfill'
```

Comment:
if these are no longer needed ( maybe there used in test ? ) you should also remove them from the package file using yarn